[BB-2450] Fix server error

0x29a opened the pull request:
This is a follow-up PR with fix for https://github.com/open-craft/opencraft/pull/591. It adds fix for notifications endpoint and message for `Status & Notification` page when user has no instance.

**JIRA tickets**:
- [BB-2450](https://tasks.opencraft.com/browse/BB-2450)

**Dependencies**: None

**Screenshots**:
![Screenshot_2020-07-22_19-34-37](https://user-images.githubusercontent.com/18251194/88209638-ab860a00-cc5b-11ea-94cb-ac274d5c65be.png)

**Testing instructions**:

1. Register, but don't verify email.
2. Go to `Status & Notifications` page.
3. You should see message like on screenshot.

**Reviewers**
- [x] @Agrendalath
- [ ] @xitij2000

Agrendalath commented on frontend/src/console/components/NoticeBoard/displayMessages.ts:
```diff
@@ -29,6 +29,12 @@ const messages = {
+    defaultMessage:
+      'There are no notifications available because your instance has not yet been created.',
+    description:
+      'Message to users who have not activated their email and therefore have no instance.'
```
Usually it's called "confirming" an email :)

Agrendalath commented on frontend/src/console/components/NoticeBoard/NoticeBoard.tsx:
```diff
@@ -180,6 +181,19 @@ const NoticeBoardComponent: React.FC<Props> = (props: Props) => {
+    <p>
+      <WrappedMessage id="noActiveInstance" messages={messages} />
+    </p>
+  );
```
Could we add a test for this?

0x29a commented:
Added test and fixed description. Also, I had to add new argument to `setupComponentForTesting` because dispatching of action in `useEffect` of `NoticeBoard` component somehow was affecting store in other test.

Hash of commit before squash: cb5260a2a49299a6dc5fb851808a9fcbd024349c.
